src/windowmanagement/HelpMenu.tsx

```diff
@@ -1,26 +1,144 @@
-import { Button, Menu } from '@mantine/core';
+import { Anchor, Button, Grid, Image, Menu, Modal, Text, Title } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import {
+  BookIcon,
+  ConsoleIcon,
+  ExitAppIcon,
+  FeedbackIcon,
+  InformationCircleOutlineIcon,
+  OpenInBrowserIcon,
+  RouteIcon
+} from '@/icons/icons';
+import { useAppSelector } from '@/redux/hooks';
 
 export function HelpMenu() {
+  const [opened, { open, close }] = useDisclosure(false);
+  const luaApi = useOpenSpaceApi();
+  const openSpaceVersion = useAppSelector((state) => state.version.openSpaceVersion);
+
+  async function toggleLuaConsole() {
+    if (!luaApi) {
+      return;
+    }
+    const visible = (await luaApi.propertyValue('LuaConsole.IsVisible')) as
+      | boolean
+      | undefined;
+    luaApi.setPropertyValueSingle('LuaConsole.IsVisible', !visible);
+  }
+
+  async function openGuiInBrowser() {
+    if (!luaApi) {
+      return;
+    }
+    const portProperty = (await luaApi.propertyValue('Modules.WebGui.Port')) as
+      | number
+      | undefined;
+    const port = portProperty ?? 4670; // TODO: Change this to correct port when hosted
+    const addressProperty = (await luaApi.propertyValue('Modules.WebGui.Address')) as
+      | string
+      | undefined;
+    const address = addressProperty ?? 'localhost';
+
+    const link = `http://${address}:${port}`;
+    window.open(link, '_blank');
+  }
+
+  function toggleShutdown() {
+    luaApi?.toggleShutdown();
+  }
+
+  function osVersionNumber(): React.JSX.Element {
```

Comment:
If this function only returns the string we don't need to return a function with JSX